rasa_core/policies/ensemble.py

```diff
@@ -335,6 +335,7 @@ def probabilities_using_best_policy(self,
         max_confidence = -1
         best_policy_name = None
         best_policy_priority = -1
+        best_policy_index = -1
```

Comment:
maybe we can clean this up together -- I just added the `best_policy_priority` when I added that feature, but perhaps should have questioned my adding of the variable instead of changing the one already there. `best_policy_name`, `best_policy_priority` and `best_policy_index` are all coming from the same thing -- could we perhaps just keep track of the `best_policy` and then get the name and priority from that? (the index is already in the name)